tikv-client/src/main/java/com/pingcap/tikv/types/BitType.java

```diff
@@ -78,4 +80,18 @@ protected Object decodeNotNull(int flag, CodecDataInput cdi) {
   public boolean isUnsigned() {
     return true;
   }
+
+  @Override
+  public Object getOriginDefaultValueNonNull(String value, long version) {
+    Long result = 0L;
+    byte[] bytes = Base64.getDecoder().decode(value);
+    if (bytes.length <= 0 || bytes.length > 8) {
+      throw new CastingException("byte[] to Long Overflow");
```

Comment:
better to say "bit type to long overflow"?

Reply:
done